_posts/en/newsletters/2019-07-31-newsletter.md

```diff
@@ -0,0 +1,211 @@
+      Bitcoin Core only disables serving bloom filters by default.  It
+      doesn't remove the feature.  Authors of wallets depending on bloom
+      filter support can easily operate their own nodes, and they can
+      also try to persuade other node operators who aren't worried about
+      the DoS vulnerability to set the configuration option to true,
+      e.g.  `peerbloomfilters=1`.
+
+## Bech32 sending support
+
+*Week 20 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% include specials/bech32/20-percentage-loss.md %}
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments of time to help curious or confused users.  In
+this monthly feature, we highlight some of the top voted questions and
+answers made since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [BIPs #800][] updates [BIP174][] to note that wallets must only sign
+  with the signature hash (sighash) type requested in a Partially-Signed
+  Bitcoin Transaction (PSBT) if the wallet consider that sighash type to
+  provide acceptable security.  Otherwise, wallets should default to
+  signing with `SIGHASH_ALL`.
+
+- [BIPs #766][] assigns [BIP155][] to the `addr` v2 proposal described
+  in [Newsletter #37][].  This proposal will allow nodes to share their
+  Tor hidden service (onion) version 3 addresses with each other, as
+  well as provide compatibility with other network address protocols.
+
+- [BIPs #643][] adds [BIP301][] about blind merge mining as previously
+  [discussed][drivechain discussion] on the Bitcoin-Dev mailing list in
+  late 2017.  Blind merge mining is intended to be used with the type of
+  decentralized sidechains called "drivechains".  Similar to other
+  decentralized sidechain proposals, drivechains have SPV security,
+  meaning miners with enough hashrate are able to steal funds from the
+  users of the sidechain.
+
+- [C-Lightning #2842][] triggers a reconnection attempt instead of
+  failing a channel when a "sync error" is received from a channel peer.
+  This is a deliberate decision to deviate from the [LN
+  specification][bolt1 error] because LND nodes seem to be generating
+  this error quite frequently and C-Lightning maintainers are worried
+  that if C-Lightning properly fails channels "at this rate, we risk
+  bifurcating the network."
```

Comment:
I'm curious where you got this quote from. I can't find it in the PR or code comments.

Reply:
Commit message here: https://github.com/ElementsProject/lightning/commit/6c7a45623e6c2367315f8db58f8faf0c47592943

Reply:
Thanks, although I think this quote is better: https://github.com/ElementsProject/lightning/pull/2842/files#diff-b50f49d55ecf2a4371d37f5817334a8eR8 😁